package.json

```diff
@@ -3,7 +3,7 @@
   "engines": {
     "node": ">=16"
   },
-  "version": "7.10.42",
+  "version": "7.10.43",
```

Comment:
Could you bump the version in a separate commit? Makes it easier to cherry-pick things later